.expeditor/scripts/end_to_end/SupervisorRunner.cs

```diff
@@ -17,6 +18,10 @@ public class SupervisorRunner {
         proc.StartInfo.RedirectStandardOutput = true;
         proc.StartInfo.RedirectStandardError = true;
         proc.StartInfo.FileName = "hab";
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
```

Comment:
Do we need / have the equivalent of `$IsLinux`?

Reply:
Yeah, it would be more clear if this had an `else`. Added in [0edbb5](https://github.com/habitat-sh/habitat/pull/7807/commits/0edbb5009c4d62a814b3e90fd901a1af8ceb2e36)

Reply:
This is a good change (and I hadn't even noticed it at first :sweat_smile: ), but I was thinking this conditional would be something we could formally set up to use in the same way that we use `$IsLinux`. That is, perhaps we should have an `$IsWindows` to complement it.

Reply:
Gotcha, this is declaring a type in `C#` which we pull into powershell [here](https://github.com/habitat-sh/habitat/blob/0edbb5009c4d62a814b3e90fd901a1af8ceb2e36/.expeditor/scripts/end_to_end/run_e2e_test_core.ps1#L95). `$IsLinux` and `$IsWindows` are both powershellisms. Maybe @mwrock knows a better way to express this?

Reply:
Ooooh, gotcha. Maybe not a big issue, then.

Feel free to ignore this :sweat_smile: